src/qt/addressbookpage.cpp

```diff
@@ -46,12 +46,13 @@ class AddressBookSortFilterProxyModel final : public QSortFilterProxyModel
 
         auto address = model->index(row, AddressTableModel::Address, parent);
 
-        if (filterRegExp().indexIn(model->data(address).toString()) < 0 &&
-            filterRegExp().indexIn(model->data(label).toString()) < 0) {
-            return false;
-        }
-
-        return true;
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 12, 0))
+        const auto pattern = filterRegularExpression().pattern();
```

Comment:
Considering https://github.com/bitcoin-core/gui/pull/592/files#r878677419, I think the only required change is:
```cpp
#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
    const auto pattern = filterRegularExpression()
#else
    const auto pattern = filterRegExp();
#endif
```

Reply:
@promag 

Unfortunately, it does not work [for me](https://github.com/hebasto/gui/commits/220521-wc-demo) (

Reply:
Thank you! [Updated.](https://github.com/bitcoin-core/gui/pull/593#issuecomment-1133658920)